### PR TITLE
stub -> app

### PR DESCRIPTION
**Pull request by erikbern**
This updates `stub` -> `app` and `Stub` -> `App`

Also adds a note to all App instantiations explaining the name change

### CODE REVIEW

File: misc/news_summarizer.py [hunk 1]
```diff
@@ -79,11 +81,11 @@ class NYArticle:
 
 
 # In order to connect to the NYT API, you will need to sign up at [NYT Developer Portal](https://developer.nytimes.com/),
-# create an Stub then grab an API key. Then head to Modal and create a [Secret](https://modal.com/docs/guide/secrets) called `nytimes`.
```
Comment (erikbern) [inline misc/news_summarizer.py:82]: lol this was ungrammatical before, but now is right

File: 02_building_containers/import_sklearn.py [hunk 2]
```diff
@@ -9,28 +9,28 @@
 
 import modal
 
-# Next, define an stub, with a custom image that installs `sklearn`.
```
Comment (erikbern) [inline 02_building_containers/import_sklearn.py:12]: lol this was ungrammatical before, but now is right (2)

File: 06_gpu_and_ml/dreambooth/dreambooth_app.py [hunk 3]
```diff
@@ -516,14 +518,14 @@ async def background():
 #
 # - `modal run dreambooth_app.py` will train the model. Change the `instance_example_urls_file` to point to your own pet's images.
 # - `modal serve dreambooth_app.py` will [serve](https://modal.com/docs/guide/webhooks#developing-with-modal-serve) the Gradio interface at a temporary location. Great for iterating on code!
-# - `modal shell dreambooth_app.py` is a convenient helper to open a bash [shell](https://modal.com/docs/guide/developing-debugging#stubinteractive_shell) in our image. Great for debugging environment issues.
+# - `modal shell dreambooth_app.py` is a convenient helper to open a bash [shell](https://modal.com/docs/guide/developing-debugging#appinteractive_shell) in our image. Great for debugging environment issues.
```
Comment (mwaskom) [inline 06_gpu_and_ml/dreambooth/dreambooth_app.py:521]: Is this going to work?

**Review by mwaskom — APPROVED** [review]
Should the comment use capitalized terms? I.e. what's important is changing the name of the class, not the variable.

File: 06_gpu_and_ml/dreambooth/dreambooth_app.py (hunk 3 above)
Comment (erikbern) [reply 06_gpu_and_ml/dreambooth/dreambooth_app.py:521]: i think it's actually already broken – the link should be https://modal.com/docs/guide/developing-debugging#interactive_shell

will update, thanks!

**Comment by erikbern** [comment]
> Should the comment use capitalized terms? I.e. what's important is changing the name of the class, not the variable.

no strong feelings, happy to update it

might merge this one first though since i already made the same change and merged it for docs

**Comment by charlesfrye** [comment]
I agree that we should update the comments to use the capital letter!

**Pull request merged**